src/main.css

```diff
@@ -2294,4 +2294,7 @@
   .project-card code {
     background: #333 !important;
   }
+  .CircleBadge {
```

Comment:
this is not a good idea, cause now you overriden ALL circlebadges fixing 1 and breaking 100

Reply:
Is it good to start with an Action-specific DOM?

Reply:
```css
       // fix gh circlebadge bg e.g. https://github.com/marketplace/dependabot-preview
        .CircleBadge--github::after { background-color: var(--color-icon-primary) }
        }

        .CircleBadge-icon[alt*="workflow logo"] {
            filter: invert(0.95) brightness(1.5) grayscale(1)
            mix-blend-mode: lighten
        }

        // fix GH circlebadge https://github.com/login/oauth/authorize?client_id=c602a8bd54b1e774f864&scope=repo
        .CircleBadge[style*="background-color: #fff; overflow"] { background-color: var(--color-bg-tertiary) }
```

These are my overrides in the style Im developing separately for GitHub,

It should fix the circlebadges that matter without breaking the majority.